lib/VerifiableCredentialStore.js

```diff
@@ -10,7 +10,11 @@ export default class VerifiableCredentialStore {
   constructor({edv, invocationSigner}) {
     this.edv = edv;
     this.invocationSigner = invocationSigner;
-    this.edv.ensureIndex({attribute: ['meta.issuer', 'content.type']});
+    this.edv.ensureIndex({attribute: [
+      'meta.issuer',
+      'meta.displayable',
+      'content.type'
+    ]});
```

Comment:
This will be a breaking change. I'm not sure if that's ok or not -- or what we used this index before in the past. But we will no longer be able to query VCs from a particular issuer with a particular type -- without knowing their displayable status or doing two queries. This is ok for now (probably?) but it may become a problem in the future.

Reply:
Thanks for pointing this out.  I'll release this as breaking.  And the good news is we currently have no need to apply this to an existing wallet.  Are there any other breaking changes along these lines in the pipe?

Reply:
^That's a question for everyone else, I don't know the answer.